tailwind.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-  extend: {},
+    extend: {},
```

Comment:
nitpick(nb, it doesn't hurt anyone): We should lint this so we get standard indentations.